301-multi-vmss-linux/nestedtemplates/base.json

```diff
@@ -27,38 +24,39 @@
   "variables": {
     "imagePublisher": "Canonical",
     "imageOffer": "UbuntuServer",
-    "imageVersion": "15.10",
+    "osVersion": "16.04.0-LTS",
```

Comment:
Consider eliminating variables like these that hold values that the schemas could be suggesting allowedValues for